test_n13_images.py

```diff
@@ -0,0 +1,28 @@
+import filecmp
+import os
+import pytest
+
+
+from requests import get
+
+
+@pytest.mark.parametrize('test_format,test_image', [
+    ('png', ['pig_png.png', 'downloaded.png']),
+    ('webp', ['wolf_webp', 'downloaded']),
+    ('svg+xml', ['svg_svg.svg', 'downloaded.svg'])
```

Comment:
ughhh, overcomplicating things.
remove the downloaded params.
just have 2 params: 
```python
@pytest.mark.parametrize('test_format,test_image', [
 +    ('png', 'pig_png.png'),
 +    ('webp', 'wolf_webp'),
 +    ('svg+xml', 'svg_svg.svg')
```
also filenames should have more consistent format:
E.g.
```wolf.webp
pig.png
svg_logo.svg
```